Documentation/network/bgp-control-plane-operation.rst

```diff
@@ -67,3 +67,61 @@ all ongoing connections with the endpoints on the failed node, and connections
 forwarded to a different node than before, will be reset.
 
 .. _Resilient Hashing: https://www.juniper.net/documentation/us/en/software/junos/interfaces-ethernet-switches/topics/topic-map/switches-interface-resilient-hashing.html
+
+Peering Link Down
+-----------------
+
+If the peering link between the BGP peers goes down, usually, both the BGP
+session and datapath connectivity will be lost. However, there may be a period
+during which the datapath connectivity is lost while the BGP session remains up
+and routes are still being advertised. This can cause the BGP peer to send
+traffic over the failed link, resulting in dropped packets. The length of this
+period depends on which link is down and the BGP configuration.
+
+If the link directly connected to the Node goes down, the BGP session will
+likely be lost immediately because the Linux kernel detects the link failure
+and shuts down the TCP session right away. If a link not directly connected to
+the Node goes down, the BGP session will be lost after the hold timer expires,
+which is set to 90 seconds by default.
+
+Mitigation
+~~~~~~~~~~
+
+To make link detection failure fast, you can adjust ``holdTimeSeconds`` and
+``keepAliveTimeSeconds`` in the BGP configuration to the shorter value.
+However, the minimal possible value is ``holdTimeSeconds=3`` and
+``keepAliveTimeSeconds=1``. The general approach to make it much faster is to
+use BFD (Bidirectional Forwarding Detection), but currently, Cilium does not
+support it.
+
+Cilium Operator Down
+--------------------
+
+If the Cilium Operator goes down, PodCIDR allocation by IPAM, and LoadBalancer
+IP allocation by LB-IPAM are stopped. Therefore, the new advertisement and
+withdraw of the PodCIDR and Service VIP routes will be stopped as well.
+
+Mitigation
+~~~~~~~~~~
+
+There's no direct mitigation in terms of the BGP. Kubernetes will eventually
+restart the Cilium Operator, and the routes will be advertised and withdrawn as
+usual.
```

Comment:
Maybe we  should mention here that running more than 1 operator replicas may help to mitigate this?